cfep-15.md

```diff
@@ -0,0 +1,40 @@
+<table>
+<tr><td> Title </td><td> Deprecate Python 2.7 </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Matthew R. Becker &lt;becker.mr@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Mar 9, 2020</td></tr>
+<tr><td> Updated </td><td> Mar 9, 2020</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Python 2.7 is officially deprecated. `conda-forge` will follow suit.
+
+## Motivation
+
+Python 2.7 has been officially deprecated by the upstream developers and Anaconda Inc.
+Further, vs2008, used to build Python 2.7, is not supported on Azure. Thus we will
+remove vs2008 in this CFEP as well.
+
+## Specification
+
+- We will remove Python 2.7 from the global pinnings.
+- We will post a notice in the documentation that Python 2.7 is deprecated.
+- We will provide an admin command that will add back Python 2.7 to any repo
+  on an opt-in basis.
+- We will provide no ongoing support for Python 2.7 builds and any builds are
+  provided on an "as-is" basis.
+- We will remove vs2008 and deprecate appveyor as a supported CI platform in conjunction 
```

Comment:
I thought there were still issues with MSBuild. Is that not true any more?

Reply:
@isuruf?

Reply:
We'll have to go to VS2017

Reply:
So things like ( https://github.com/conda-forge/libdb-feedstock/pull/9 ) will work with VS2017?

Reply:
For background we have to deprecate appveyor to implement cfep-13 which has already passed. So that means removing vs2008 no matter what. Appveyor does support secrets per feedstock. The keys they use to encrypt secrets are shared across conda-forge.

Reply:
No complaints with deprecating vs2008. The question is some recipes require MSBuild and they only work on AppVeyor (not Azure). Though maybe @isuruf is saying that is not the case?

Reply:
> So things like ( conda-forge/libdb-feedstock#9 ) will work with VS2017?

It might or might not. Dunno.

Reply:
```suggestion
- We will remove vs2008 as a compiler in conjunction 
```